addons/html_editor/static/src/main/power_buttons_plugin.js

```diff
@@ -66,9 +66,10 @@ export class PowerButtonsPlugin extends Plugin {
             )
         ) {
             this.powerButtons.classList.remove("d-none");
-            let direction = block.getAttribute("dir");
-            if (block.tagName === "LI") {
-                direction = block.parentElement.getAttribute("dir");
+            let direction = block.getAttribute("dir") || this.editable.getAttribute("dir");
+            if (element.parentElement.tagName === "LI") {
+                direction =
+                    closestElement(element, "ul[dir], ol[dir]")?.getAttribute("dir") || direction;
```

Comment:
Why not look for `closest[dir]` ? Wouldn't that solve it for all cases? ul, ol, block, _and_ `this.editable`?

Reply:
Indeed we can do that. I have updated it.